# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/unit/xpcshell.ini
# Commit: 942a92874526
# Full Hash: 942a92874526e7a18167af89e358d69cbaeab268
# Author: Boris Zbarsky <bzbarsky@mit.edu>
# Date: 2020-02-22 09:52:08
# Regressor Bug: 267645
# File Overlap Count: 1
# Description:
#   Bug 267645 part 1.  Add a way to ask for DOMException to be exposed in sandboxes. r=bholley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63278
# ==============================================================================

diff -r d21671811dff -r 942a92874526 js/xpconnect/tests/unit/xpcshell.ini
--- a/js/xpconnect/tests/unit/xpcshell.ini	Tue Feb 18 11:14:49 2020 +0000
+++ b/js/xpconnect/tests/unit/xpcshell.ini	Sat Feb 22 04:56:52 2020 +0000
@@ -108,6 +108,7 @@
 [test_subScriptLoader.js]
 [test_rewrap_dead_wrapper.js]
 [test_sandbox_metadata.js]
+[test_sandbox_DOMException.js]
 [test_exportFunction.js]
 [test_promise.js]
 [test_returncode.js]
